test/java/build.gradle

```diff
@@ -0,0 +1,44 @@
+apply plugin: 'java-library'
```

Comment:
Keep quotation marks consistent - apprently `"` is the way to go.
See update at: 
https://github.com/juliaogris/sysl/blob/json_serializer_mv/test/java/build.gradle

Reply:
This is still missing.